.github/workflows/ci.yml

```diff
@@ -68,11 +71,26 @@ jobs:
         run: |
           wget https://github.com/gradinac/musl-bundle-example/releases/download/v1.0/musl.tar.gz
           tar xvf musl.tar.gz
-
       - run: bin/build-native-image.sh
         env:
           CI: true
       - uses: actions/upload-artifact@master
         with:
           name: ${{ matrix.artifact }}
           path: scalafmt
+  dockerize:
```

Comment:
Will it push new image on every PR?

Reply:
yep, with a custom tag (refs/pull/2/merge | pr-2-merge)

Reply:
Can you please configure it as dedicated workflow running only on tags push? 

```
on:
  push:
    branches: [master]
    tags: ["*"]
```

Image creation on every PR would be redundant

Reply:
Maybe it is better to add this flow only for tags?

Reply:
I added condition for tags only.

Reply:
Thank you! I will test it on next release